tests/integration/targets/dcnm_vrf/tests/dcnm/merged.yaml

```diff
@@ -65,6 +65,10 @@
     - '"{{ ansible_switch2 }}" in result.diff[0].attach[1].ip_address'
     - 'result.diff[0].vrf_name == "ansible-vrf-int1"'
 
+- name: MERGED - sleep for 40 seconds for DCNM to completely update the state
```

Comment:
Here and elsewhere, let's try and avoid arbitrary sleep statements like this and instead use the following approach:

https://github.com/CiscoDevNet/ansible-dcnm/blob/develop/tests/integration/targets/dcnm_network/tests/dcnm/self-contained-tests/sm_mcast_params.yaml#L32

Reply:
Better to check for a specific condition and loop until that condition is reached instead of hoping 40 seconds is long enough.  Also if we reach the state sooner then we move to the next task more quickly.

Reply:
Changes done as suggested